packages/nestjs/src/setup.ts

```diff
@@ -11,6 +11,7 @@ import { Catch } from '@nestjs/common';
 import { Injectable } from '@nestjs/common';
 import { Global, Module } from '@nestjs/common';
 import { APP_FILTER, APP_INTERCEPTOR, BaseExceptionFilter } from '@nestjs/core';
+import { RpcException } from '@nestjs/microservices';
```

Comment:
This breaks NestJS projects which don't use `@nestjs/microservices`.

I've tried creating an issue, but couldn't submit it because GH kept showing "Something went wrong" popup.

Reply:
Thanks for writing in! I am currently working on a fix for the nest SDK and will remove the dependency again as part of that PR. [ref](https://github.com/getsentry/sentry-javascript/pull/13278)